numpy/linalg/tests/test_regression.py

```diff
@@ -90,6 +90,12 @@ def test_svd_no_uv(self):
                 assert_equal(np.linalg.matrix_rank(a), 1)
                 assert_array_less(1, np.linalg.norm(a, ord=2))
 
+    def test_norm_object_array(self):
```

Comment:
I've added more tests in bfa3ed9d29314d9d4a06eb5172ae855d19ae7c8d.

Everything that was broken before the regression is still broken.